index.js

```diff
@@ -22,10 +22,12 @@ module.exports = {
     'eol-last':                    ['error'],
     'eqeqeq':                      ['error'],
     'indent':                      ['error', 2, {SwitchCase: 1}],
+    'key-spacing':                 ['error', {beforeColon: false, afterColon: true}],
```

Comment:
Personally think it's a bit draconian to prevent alignment of object values (like in this file for instance)